src/errors.ts

```diff
@@ -13,6 +13,170 @@
 // limitations under the License.
 import {ProsopoEnvError} from "@prosopo/contract";
 
+export const ERRORS = {
+    GENERAL: {
```

Comment:
As mentioned on another PR, I think there's a case for a common / shared repo to house all this stuff under one roof

Reply:
I think our `i18n` package is the shared place. This errors.ts file is probably not even needed. I'll investigate.